src/engine/Join.cpp

```diff
@@ -739,7 +752,7 @@ ad_utility::AddCombinedRowToIdTable Join::makeRowAdder(
       1,
       IdTable{getResultWidth(), allocator()},
       cancellationHandle_,
-      keepJoinColumn_,
+      _keepJoinColumn,
```

Comment:
If the variable is named with the underscore at the end, this change disappears.